### PR TITLE
Subscription orders improvements

### PR DESCRIPTION
**Pull request by shahbaz-mehar**

### CODE REVIEW

**Review by BushraAsif — COMMENTED** [review]
Changes reviewed

File: altapay.php [hunk 1]
```diff
@@ -371,23 +371,24 @@ function altapay_meta_box_side( $post_or_order_object ) {
 				if ( $status === 'released' ) {
 					echo '<strong>' . __( 'Payment Released.', 'altapay' ) . '</strong>';
 				} else {
-					$charge = $reserved - $captured - $refunded;
-					if ( $charge <= 0 ) {
-						$charge = 0.00;
-					}
+					$charge        = max( 0, $reserved - $captured - $refunded );
```
Comment (emicha) [inline altapay.php:374]: Are you sure about this?

What if the merchant wants to charge the remaining amount to be captured?

Reserved - 1000 (2 pairs of shoes)
Captured - 500 (1 pair of shoes)
Refunded - 500 (1 pair of shoes) - Did not like model

Charge: 500 (remaining pair of shoes)

Comment (shahbaz-mehar) [reply altapay.php]: @emicha I have updated the calculations.

File: altapay.php [hunk 2]
```diff
@@ -371,23 +371,24 @@ function altapay_meta_box_side( $post_or_order_object ) {
 				if ( $status === 'released' ) {
 					echo '<strong>' . __( 'Payment Released.', 'altapay' ) . '</strong>';
 				} else {
-					$charge = $reserved - $captured - $refunded;
-					if ( $charge <= 0 ) {
-						$charge = 0.00;
-					}
+					$charge        = max( 0, $reserved - $captured );
```
Comment (emicha) [inline altapay.php:374]: What is there has been refunds?

Comment (shahbaz-mehar) [reply altapay.php]: @emicha I've updated the chargeable amount calculations to include refunds as well.

Example:
Reserved = 1500
Captured = 500
Refunded = 500

Chargeable = Reserved - Captured + Refunded
Chargeable = 1500 - 500 + 500 = 1500

**Review by emicha — APPROVED** [review]

**Pull request merged**